# Exercise 2: The Living City

#### Datasets: `London.csv`, `Chicago.csv`

## Comparing London and Chicago

In [1]:
import pandas as pd

In [2]:
London = pd.read_csv("London.csv", index_col='Year')

In [3]:
London

Population  Deaths
Year                    
1840     1842458   46281
1841     1877963   45284
1842     1916860   45272
1843     1953787   48574
1844     2033816   50423
1845     2073298   48332
1846     2113535   49089
1847     2195401   60442
1848     2238703   57628
1849     2282858   68432
1850     2327884   48579
1851     2373799   55354
1852     2420619   54213
1853     2468362   61202
1854     2517048   73697

**How does London compare to a similar-sized city like Chicago?**

Here is the same kind of data for Chicago for 2010-2018.

In [4]:
Chicago = pd.read_csv("Chicago.csv", index_col='Year')

In [5]:
Chicago

Population  Deaths
Year                    
2010     2698831   18595
2011     2707123   18743
2012     2714844   18911
2013     2718789   18825
2014     2722407   19003
2015     2720556   19308
2016     2704965   19809
2017     2716462   19742
2018     2705988   19660

## Creating a Variable of Interest

It is easy to compare Chicago and London side by side because they have similar populations.

However, we still need to factor in the total population when considering death rate, also known as "mortality".

This is done with the following calculation:

$$death \ rate_{1000} = {deaths \over population} \times 1000$$



We can express this formula in our code as follows.

In [6]:
London['Deaths'] / London['Population'] * 1000

Year
1840    25.119161
1841    24.113361
1842    23.617792
1843    24.861461
1844    24.792312
1845    23.311651
1846    23.226017
1847    27.531189
1848    25.741691
1849    29.976459
1850    20.868308
1851    23.318739
1852    22.396337
1853    24.794580
1854    29.279140
dtype: float64

That's nice, but the data is more useful all together. (And it looks nicer besides!)

In [7]:
London['Deaths per 1000'] = London['Deaths'] / London['Population'] * 1000

In [8]:
London

Population  Deaths  Deaths per 1000
Year                                     
1840     1842458   46281        25.119161
1841     1877963   45284        24.113361
1842     1916860   45272        23.617792
1843     1953787   48574        24.861461
1844     2033816   50423        24.792312
1845     2073298   48332        23.311651
1846     2113535   49089        23.226017
1847     2195401   60442        27.531189
1848     2238703   57628        25.741691
1849     2282858   68432        29.976459
1850     2327884   48579        20.868308
1851     2373799   55354        23.318739
1852     2420619   54213        22.396337
1853     2468362   61202        24.794580
1854     2517048   73697        29.279140

**How did the mortality rate of 19th Century London compare to 21st Century Chicago?**

For London, we wrote:

    London['Deaths'] / London['Population'] * 1000
    
Can you modify the code above to calculate the mortality rate **for Chicago**?

We combined these data with the rest of our London data with the following:

    London['Deaths per 1000'] = London['Deaths'] / London['Population'] * 1000

(And then we simply wrote `London` to *see* our data):

    London

What would that look like for Chicago as well?

## Interpretation

London experience two major cholera outbreaks between 1840 and 1854. **Can you figure out which years?**

**What methods did you use to pick these years? What did you have to consider?**

…